docs/book/getting-started/zenml-pro/README.md

```diff
@@ -1,40 +1,38 @@
+Learn more about ZenML Pro on the [ZenML Website](https://zenml.io/pro).
 
-We have built the ZenML Pro experience from the ground-up. With ZenML Pro, you get
-access to a new dashboard, with a better experience. The new dashboard features
-more functionality such as
-the [Model Control Plane](../../user-guide/starter-guide/track-ml-models.md)
-and [Artifact Control Plane](../../user-guide/starter-guide/manage-artifacts.md).
+{% hint style="info" %}
+If you're interested in assessing ZenML Pro, you can simply create
+a [free account](https://cloud.zenml.io/?utm\_source=docs\&utm\_medium=referral\_link\&utm\_campaign=cloud\_promotion\&utm\_content=signup\_link).
+Learn more about ZenML Pro on the [ZenML Website](https://zenml.io/pro).
+{% endhint %}
 
-### Run templates for running your pipelines from the dashboard or the API
+## Deployment scenarios: SaaS vs Self-hosted
 
-ZenML Pro enables you to [create and run templates](../../how-to/trigger-pipelines/README.md#run-templates).
-This way, you can use the dashboard or our Client/REST API to run a pipeline with updated configuration
-which allows you to iterate quickly with minimal friction. 
+One of the most straightforward paths to start with a deployed ZenML server is
+to use [the SaaS version of ZenML Pro](https://zenml.io/pro). The ZenML Pro offering eliminates the need
+for you to dedicate time and resources to deploy and manage a ZenML server,
+allowing you to focus primarily on your MLOps workflows.
 
-### Triggers, CI/CD, Reports and more
+However, ZenML Pro can also be deployed fully self-hosted. Please [book a demo](https://www.zenml.io/book-your-demo)
+to learn more.
 
-Additionally, ZenML Pro users get exclusive access to an array of
-cloud-specific features, such as triggers, integrating with your code
-repository CI/CD system, generating usage reports and more.
+Click [here](../system-architectures.md) to understand about the various deployment scenarios with ZenML Pro.
 
-Learn more about ZenML Pro on the [ZenML Website](https://zenml.io/pro).
+<table data-card-size="large" data-view="cards"><thead><tr><th></th><th></th><th data-hidden></th><th data-hidden data-type="content-ref"></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td><mark style="color:purple;"><strong>Tenants</strong></mark></td><td>Tenants in ZenML Pro</td><td></td><td></td><td><a href="./tenants.md">tenants.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Organizations</strong></mark></td><td>Organizations in ZenML Pro</td><td></td><td></td><td><a href="./organization.md">organization.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Teams</strong></mark></td><td>Teams in ZenML Pro</td><td></td><td></td><td><a href="./teams.md">teams.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Roles</strong></mark></td><td>Roles in ZenML Pro</td><td></td><td></td><td><a href="./roles.md">roles.md</a></td></tr></tbody></table>
```

Comment:
Feels like there is too much redirection here. The table should stay I guess, but then the paragraph itself seems to send you in lots of directions.